startup_runners/Makefile

```diff
@@ -19,14 +19,14 @@ startup_runner.class: startup_runner.java
 
 
 outer_startup_runner_c: outer_startup_runner.c
-	${CC} ${C_ITER_RUNNER_CFLAGS} ${CFLAGS} -L${PWD}/../krun/libkrun \
-		${CPPFLAGS} outer_startup_runner.c \
+	${CC} --std=c99 ${C_ITER_RUNNER_CFLAGS} ${CFLAGS} -L${PWD}/../krun/libkrun \
```

Comment:
It works fine here, and I'd prefer not to use a vendor-specific standard unless we really have to.